source/clear-linux/get-started/virtual-machine-install/virtualbox-cl-installer.rst

```diff
@@ -1,258 +1,384 @@
-
+virtual hard disk where |CL| has been installed.
 
-#. Power off the |CL| VM.
+#. In case the |CL| VM is not off, select :guilabel:`Machine>ACPI Shutdown`.
 
-   .. image:: ./figures/vbox/vbox-shutdown-vm.png
+   .. figure:: figures/vbox/virtualbox-cl-installer-11.png
+      :scale: 100%
       :alt: Powering off a VirtualBox VM
 
-#. Click *Settings* to configure the |CL| VM.
+      Figure 11: Powering off a VirtualBox VM
 
-   .. image:: ./figures/vbox/vbox-vm-created.png
-      :alt: A VM selected in VirtualBox Manager
+#. Click *Settings* to configure the |CL| VM.
 
-#. From the *VM - Settings* window, navigate to the *Storage* pane from the 
+#. From the *VM - Settings* window, navigate to the *Storage* pane from the
    left-hand side.
 
-#. From the middle *Storage Devices* column, click the blue CD disk labeled  
-   *clear-<VERSION>-installer.iso* under the *Controller: IDE* from.
+#. From the middle *Storage Devices* column, click the blue CD disk labeled
+   *clear-<VERSION>-live-desktop.iso* under the *Controller: IDE* from.
 
-#. From the right-hand *Attributes* column, click the blue CD disk next to 
-   the *Optical Drive* drop down menu and click *Remove Disk from Virtual 
+#. From the right-hand *Attributes* column, click the blue CD disk next to
+   the *Optical Drive* drop down menu and click *Remove Disk from Virtual
    Drive*
 
-   .. image:: ./figures/vbox/vbox-vm-settings-unmount-ISO.png
-      :alt: Unmounting an ISO in VirtualBox VM Settings        
+   .. figure:: figures/vbox/virtualbox-cl-installer-12.png
+      :scale: 100%
+      :alt: Remove Disk from Virtual Drive
+
+      Figure 12: Remove Disk from Virtual Drive
 
-#. Click *OK* to exit the *VM Settings* menu and return to the main 
+#. Click *OK* to exit the *VM Settings* menu and return to the main
    |VBM|.
 
+Install |VB| Linux Guest Additions
+==================================
+
+|CL| provides Linux Guest Additions drivers for full compatibility and an install script in the **kernel-lts** (Long Term Support) bundle by |CL|.
+
+#. From the |VBM| select the |CL| VM, and select :guilabel:`Start`.
+
+#. Log in.
+
+#. Open a Terminal.
+
+#. Validate the installed kernel is **kernel-lts** by checking the output
+   of the :command:`uname -r` command. It should end in **.lts** or **.lts2018**.
 
+   .. code-block:: bash
+
+      uname -r
+      4.<VERSION>.lts
+
+   If the running kernel is not **lts**: install the LTS kernel manually,
+   update the bootloader, and check again:
 
+   .. code-block:: bash
+
+      swupd bundle-add kernel-lts
+      clr-boot-manager set-kernel $(basename $(realpath /usr/lib/kernel/default-lts))
+      clr-boot-manager update
+      reboot
 
-.. include:: virtualbox.rst
-   :start-after: vbox-start-vm-and-lga-begin:
-   :end-before: vbox-start-vm-and-lga-end:
+.. todo: What to do if it ends in `lts2018`?
 
+#. Remove any kernel bundles that are not *kernel-lts* or *kernel-install*
```

Comment:
@puneetse When I tested this, another kernel ending in `lts2018` showed up. 
Regardless of whether that's a ref only, it needs to be addressed. How do you want to handle that?  
Add it to your exception list? e.g.,`Remove any kernels that are not...`

Reply:
kernel-lts is a virtual bundle that includes the latest LTS kernel bundle, which at this time is kernel-lts-2018. So the installer should continue to install the kernel-lts bundle, but when the user looks, they'll see the kernel-lts-2018 bundle is installed. They'll eventually get the kernel-lts-2019 bundle also, automatically, once it exists and the kernel-lts bundle is changed to include it instead.